server/datastore/mysql/carves.go

```diff
@@ -28,12 +29,14 @@ func (d *Datastore) NewCarve(metadata *kolide.CarveMetadata) (*kolide.CarveMetad
 		?,
 		?,
 		?,
+		?,
 		?
 	)`
 
 	result, err := d.db.Exec(
 		stmt,
 		metadata.HostId,
+		metadata.CreatedAt.Format(mySQLTimestampFormat),
```

Comment:
Because I need a consistent timestamp to generate file keys in S3, this ensures that the creation time is the same across in both the carve name and the associated metadata. Also, this allows me to start the multipart upload before inserting the new metadata entry in the DB.